go.mod

```diff
@@ -1,19 +1,16 @@
 module github.com/containerd/ttrpc
 
-go 1.19
+go 1.22
```

Comment:
This is mainly to keep the dependencies up-to-date.

e.g., for google.golang.org/grpc
v1.69.2 requires 1.22
v1.67.3 requires 1.21

Let me know if there is any concern.